client/src/hoc/Wrapper.tsx

```diff
@@ -26,7 +27,9 @@ const Wrapper = (Component: NextComponentType) => {
       <ThemeProvider theme={theme}>
         <CssBaseline />
         <Layout>
-          <Component {...props} />
+          <AnimatePresence exitBeforeEnter>
+            <Component {...props} />
+          </AnimatePresence>
```

Comment:
nah not working
this wrapper is really bad design
gotta find another workaround for this redux crap to start working properly in the _app